packages/dev/docs/pages/releases/2022-03-07.mdx

```diff
@@ -0,0 +1,226 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: This month we have been concentrating on updating our pre-released components along with some improvements to our Slider hooks and general bug fixes. Thank you to all those who reported issues and contributed fixes in this release!
+date: 2022-03-07
+---
+
+# March 7, 2022 Release
+
+This month we have been concentrating on updating our pre-released components along with some improvements to our Slider hooks and general bug fixes. Thank you to all those who reported issues and contributed fixes in this release!
+
+## New features
+- Allow custom messages on `DismissButton` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2868)
+- Adding `name` to `ComboBox` props - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2880)
+## Fixes
+- Fire `onChangeEnd` for PageUp/PageDown/Home/End in `useSliderThumb` - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/2661)
+- Fix `TableView` rendering of rows with ids of 0 - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2840)
+- Update `Tab` index when selection is changed and not focused - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2829)
+- Fix controlled `NumberField` showing old value on blur - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2882)
+- Fix Chrome warning for `userAgent` and `platform` usage - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/2612)
+- Remove unused `pageSize` from `SliderProps` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2917)
```

Comment:
I believe the `pageSize` was something added after the last release so no need to mention here. 
<img width="1259" alt="image" src="https://user-images.githubusercontent.com/9661127/157123167-97d0a2d5-ad21-4843-9679-5078ff592928.png">